From 932d0f05720df89eacae00ec06984d2286cb4258 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Fri, 31 May 2013 12:02:14 +0900
Subject: [PATCH 528/675] LTSI: ARM: shmobile: r8a7778: keep TMU resource for
 LTSI

LTSI TMU on r8a7778 should keep its resource (= non __initdata)
since these are used from early_platform_add_devices()

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
---
 arch/arm/mach-shmobile/setup-r8a7778.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-shmobile/setup-r8a7778.c b/arch/arm/mach-shmobile/setup-r8a7778.c
index 36e7572..3146306 100644
--- a/arch/arm/mach-shmobile/setup-r8a7778.c
+++ b/arch/arm/mach-shmobile/setup-r8a7778.c
@@ -58,24 +58,24 @@ static struct plat_sci_port scif_platform_data[] __initdata = {
 };
 
 /* TMU */
-static struct resource sh_tmu0_resources[] __initdata = {
+static struct resource sh_tmu0_resources[] = {
 	DEFINE_RES_MEM(0xffd80008, 12),
 	DEFINE_RES_IRQ(gic_iid(0x40)),
 };
 
-static struct sh_timer_config sh_tmu0_platform_data __initdata = {
+static struct sh_timer_config sh_tmu0_platform_data = {
 	.name			= "TMU00",
 	.channel_offset		= 0x4,
 	.timer_bit		= 0,
 	.clockevent_rating	= 200,
 };
 
-static struct resource sh_tmu1_resources[] __initdata = {
+static struct resource sh_tmu1_resources[] = {
 	DEFINE_RES_MEM(0xffd80014, 12),
 	DEFINE_RES_IRQ(gic_iid(0x41)),
 };
 
-static struct sh_timer_config sh_tmu1_platform_data __initdata = {
+static struct sh_timer_config sh_tmu1_platform_data = {
 	.name			= "TMU01",
 	.channel_offset		= 0x10,
 	.timer_bit		= 1,
-- 
1.8.2

